services/movie_session.py

```diff
@@ -42,3 +42,8 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    movie_session = MovieSession.objects.get(pk=movie_session_id)
```

Comment:
The `get_taken_seats` function should include exception handling for cases where the movie session with the specified ID does not exist. Consider using a try-except block to catch `MovieSession.DoesNotExist` exceptions.